### PR TITLE
Playwright: replace build configuration objects with a function.

### PR DESCRIPTION
**Pull request by worldomonation**
#### Changes proposed in this Pull Request

This PR fills in details required for the WPCOM Playwright e2e test tasks.

Key changes:
- replace the static build configuration object with a function.

### CODE REVIEW

File: .teamcity/_self/projects/WPComTests.kt [hunk 1]
```diff
@@ -207,6 +207,178 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
...
+                    # Decrypt config
+                    openssl aes-256-cbc -md sha1 -d -in ./config/encrypted.enc -out ./config/local-test.json -k "%CONFIG_E2E_ENCRYPTION_KEY%"
+
+                    # Run the test
+                    export VIEWPORT_NAME=$viewportName
+                    export LOCALE=en
+                    export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
+                    export DEBUG=pw:api
+
+                    xvfb-run yarn jest --reporters=jest-teamcity --reporters=default --testNamePattern @parallel --maxWorkers=%E2E_WORKERS% specs/specs-playwright-wpcom
```
Comment (worldomonation) [inline .teamcity/_self/projects/WPComTests.kt:300]: This is just a placeholder while we figure out how best to organize the different spec files.
More details in pciE2j-sq-p2

File: .teamcity/_self/projects/WPComTests.kt [hunk 2]
```diff
@@ -207,6 +207,178 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
 	}
 }
 
+fun gutenbergPlaywrightBuildType( viewportName: String, buildUuid: String ): BuildType {
```
Comment (worldomonation) [inline .teamcity/_self/projects/WPComTests.kt:210]: For this function I copied over existing function `gutenbergBuildType` and made some changes.

File: .teamcity/_self/projects/WPComTests.kt [hunk 3]
```diff
@@ -207,6 +207,178 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
...
+    return BuildType {
+        id("WPComTests_gutenberg_Playwright_$viewportName")
+		uuid=buildUuid
```
Comment (scinos) [inline .teamcity/_self/projects/WPComTests.kt:213]: /nit: indent is wrong

**Review by scinos — APPROVED** [review]

File: .teamcity/_self/projects/WPComTests.kt (hunk 3 above)
Comment (worldomonation) [reply .teamcity/_self/projects/WPComTests.kt:213]: Odd, it's not showing in my editor as having wrong indent :|

Comment (worldomonation) [reply .teamcity/_self/projects/WPComTests.kt:213]: There we go. Fixed in cb6566d1d3eb595565d34912d9875dbccc96dbab.

**Pull request merged**